### ISSUE TITLE
aws_iot_policy doesn't support import

### ISSUE THREAD

**Issue by magiconair** [terraform-providers/terraform-provider-aws#13703]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

I want to import an existing `aws_iot_policy` into the state but get the following error:

```
Error: resource aws_iot_policy doesn't support import
```

Version:
```
Terraform v0.12.26
+ provider.aws v2.65.0
```

### New or Affected Resource(s)

* aws_iot_policy

**Comment by magiconair** [comment]
Wow. This was fast

**Comment by bflad** [comment]
Support for `aws_iot_policy` resource import has been merged and will release with version 2.68.0 of the Terraform AWS Provider, later this week. Thanks to @ewbankkit for the implementation. 👍
